src/common/nbytes.ts

```diff
@@ -75,7 +75,7 @@ export abstract class NBytes extends Serializable {
     try {
       if (buff.length - offset < this.bsize) {
         /* istanbul ignore next */
-        throw new Error(`Buffer length must be ${this.bsize} bytes. Only have ${buff.length - offset} remaining in buffer.`);
+        throw new Error(`Error`);
```

Comment:
Could this error say `'Error - NBytes.fromBuffer: not enough space available in buffer.'` to be more descriptive.

Reply:
Resolved